packages/origin/src/test/CertificateLogic.test.ts

```diff
@@ -546,7 +539,7 @@ describe('CertificateLogic-Facade', () => {
 
         const certificate = await new Certificate.Entity('6', conf).sync();
 
-        blockCreationTime = (await web3.eth.getBlock('latest')).timestamp;
+        blockCreationTime = parseInt((await web3.eth.getBlock('latest')).timestamp.toString(), 10);
```

Comment:
What is the reason for parse(string), https://web3js.readthedocs.io/en/v1.2.1/web3-eth.html#getblock timestamp should be number already.

Reply:
Unfortunately docs aren't up to date.

![typess](https://user-images.githubusercontent.com/4950658/69229691-37540d80-0b86-11ea-81d6-e7d7cd2b8783.png)